packages/contracts/package.json

```diff
@@ -31,7 +31,9 @@
     "deploy:diamond": "ts-node ./scripts/deploy/manager/DiamondInAll.ts",
     "task": "ts-node ./scripts/task/task.ts",
     "clean": "forge clean",
-    "purge": "yarn clean"
+    "purge": "yarn clean",
+    "anvil": "ts-node ./scripts/anvil/anvil.ts",
```

Comment:
I think this is out of date from development because I updated `ts-node` to use `tsx` cause its faster.

Reply:
```suggestion
    "anvil": "tsx ./scripts/anvil/anvil.ts",
```